src/components/secondary-cta.jsx

```diff
@@ -0,0 +1,32 @@
+import React, {useState} from 'react';
+import CtaRocketship from '../images/svg-2022/cta-rocketship';
+
+
+const SecondaryCta = ({textDescription}) => {
+    const [message, setMessage] = useState('');
+
+    const clicked = async e => {
+        e.preventDefault();
+        setMessage('Copied the HBP website to clipboard!');
+        const el = document.createElement('input');
+        el.value = window.location.href;
+        document.body.appendChild(el);
+        el.select();
+        document.execCommand('copy');
+        document.body.removeChild(el);
```

Comment:
Is there not an easier way to do this? I'd do some google searching, unless you already did. If not, that's ok

Reply:
Did some google searching, and yeah, this is how they copied the current website's link